packages/postgraphile-core/src/index.ts

```diff
@@ -73,6 +73,7 @@ export interface PostGraphileCoreOptions {
   skipPlugins?: Array<Plugin>;
   jwtPgTypeIdentifier?: string;
   jwtSecret?: string;
+  jwtSignOptions?: string;
```

Comment:
jwtSignOptions is not a string, I think you should add and export another interface, eg:

```ts
export interface PostGraphileJwtSignOptions {
    audience?: ???;
    issuer?: ???;
    expiresIn?: ???;
}
```

and use it instead of the string type:

```suggestion
  jwtSignOptions?: PostGraphileJwtSignOptions; 
```

Reply:
Done